src/pages/Home/Home.page.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react';
+
+import {Helmet} from "react-helmet";
+import MaskInput from 'react-maskinput';
+
+import { useGeneralContext } from '../../state/GeneralContext';
+import useApodApi from '../../utils/hooks/useApodApi';
+import Col from '../../components/Col';
+import Button from '../../components/Button';
+import Row from '../../components/Row';
+import ImageCard from '../../components/ImageCard';
+
+function HomePage() {
+  const [ title, setTitle ] = useState("Img of the Day!");
+  const { date, setDate } = useGeneralContext();
+
+  const params = {
+    date: date,
+  };
```

Comment:
En ES6 ya no es necesario definir en objetos `key` y `value` si ambos se llaman igual, es decir que esta línea se puede simplificar a lo siguiente y seguir haciendo lo mismo.
```suggestion
  const params = {
    date,
  };
```
Con eso le dices al objeto que tome el valor de date y dentro del objeto le ponga como `key` para acceder a él, el nombre de tu variable.